esp/src/src/Utility.ts

```diff
@@ -91,10 +91,10 @@ export function espTime2SecondsTests() {
     }, this);
 }
 
-export function convertedSize (intsize: number): string {
-    const unitConversion = ["Bytes","KB","MB","GB","TB","PB","EB","ZB","YB"];
-    const x = Math.floor(Math.log(intsize)/ Math.log(1024));
-    return (intsize/ Math.pow(1024,x)).toFixed(2) + " " + unitConversion[x];
+export function convertedSize(intsize: number): string {
```

Comment:
I'm assuming your VSCode "format on save" is doing some of these format changes similar to the ones I mentioned above?